### PR TITLE
saved judges and cases functionality and styled components

### PR DESCRIPTION
**Pull request by MichelleCCodes**
## Description
- user can successfully save cases 
- there is only one success message instead of the same amount as cases saved
- header made to fit 100% screen width
- user can access pdf to the saved case
- more relevant information displayed on save judge and case table

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)

## Checklist:
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have removed unnecessary comments/console logs from my code

### CODE REVIEW

File: src/components/pages/Cases/CaseTable.js [hunk 1]
```diff
@@ -44,7 +44,6 @@ export default function CaseTable(props) {
   const { searchText, searchedColumn, selectedRowID } = state;
 
   const onSelectChange = selectedRowID => {
-    console.log('selectedRowID changed: ', selectedRowID);
```
Comment (katieolson84) [inline src/components/pages/Cases/CaseTable.js:47]: Nice job removing this console.log

File: src/components/pages/Cases/CaseTable.js [hunk 2]
```diff
@@ -267,27 +266,21 @@ export default function CaseTable(props) {
...
+
+  const findRowByID = (rowID, rowData) => {
+    for (let i = 0; i < rowData.length; i++) {
+      let currentRow = rowData[i];
+      if (rowData[i].case_id === rowID) {
+        return currentRow;
       }
-    });
+    }
+    return 'Case does not exist';
```
Comment (katieolson84) [inline src/components/pages/Cases/CaseTable.js:277]: I like the addition of this message.

File: src/components/pages/Home/RenderHomePage.js [hunk 3]
```diff
@@ -90,7 +90,7 @@ function RenderHomePage(props) {
           res.data.map(eachCase => {
             return {
               ...eachCase,
-              id: eachCase.case_number,
+              id: eachCase.case_id,
```
Comment (dgamboa) [inline src/components/pages/Home/RenderHomePage.js:93]: Great

**Review by dgamboa — APPROVED** [review]
Good clean up all around with the right case variables and better function implementations. Approved

**Pull request merged**